agg.pretty/ADP2389ACPZ-R7.kicad_mod

```diff
@@ -0,0 +1,186 @@
+  (fp_line (start -1.6 -2.5) (end -1.6 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -1.9 -2.5) (end -1.6 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -1.4 -2.55) (end -1.4 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -1.1 -2.55) (end -1.4 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -1.1 -2.5) (end -1.1 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -1.4 -2.5) (end -1.1 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.9 -2.55) (end -0.9 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.6 -2.55) (end -0.9 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.6 -2.5) (end -0.6 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.9 -2.5) (end -0.6 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.4 -2.55) (end -0.4 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.1 -2.55) (end -0.4 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.1 -2.5) (end -0.1 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -0.4 -2.5) (end -0.1 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.1 -2.55) (end 0.1 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.4 -2.55) (end 0.1 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.4 -2.5) (end 0.4 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.1 -2.5) (end 0.4 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.6 -2.55) (end 0.6 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.9 -2.55) (end 0.6 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.9 -2.5) (end 0.9 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 0.6 -2.5) (end 0.9 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.1 -2.55) (end 1.1 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.4 -2.55) (end 1.1 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.4 -2.5) (end 1.4 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.1 -2.5) (end 1.4 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.6 -2.55) (end 1.6 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.9 -2.55) (end 1.6 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.9 -2.5) (end 1.9 -2.55) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start 1.6 -2.5) (end 1.9 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -2.5 -1.23) (end -1.23 -2.5) (layer Dwgs.User) (width 0.1524))
+  (fp_line (start -2.22 -2.63) (end -2.62 -2.24) (layer F.SilkS) (width 0.15))
+  (fp_line (start -3 -3) (end 3 -3) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 3 -3) (end 3 3) (layer F.CrtYd) (width 0.01))
+  (fp_line (start 3 3) (end -3 3) (layer F.CrtYd) (width 0.01))
+  (fp_line (start -3 3) (end -3 -3) (layer F.CrtYd) (width 0.01))
+  (pad 34 smd rect (at -0.16 1.01) (size 3.340999 1.625) (layers F.Cu F.Paste F.Mask))
+  (pad 33 smd rect (at 0 -1.0115) (size 3.7 1.677) (layers F.Cu F.Paste F.Mask))
```

Comment:
The EPs should probably have smaller mask apertures otherwise too much paste will be deposited.